src/components/Filters/SchemaSieve.ts

```diff
@@ -220,6 +223,22 @@ const flattenAccumulator = (
 			}
 
 			const newKey = parentKey + delimiter + key;
+
+			// Move any required fields into flattened items in the top-level required array
+			forEach(value.required, (requiredKey) => {
```

Comment:
NIT: Use the native `forEach` instead

Reply:
@sradevski Lodash `forEach` has the nice quality of safely handling falsey values, which means we don't have to check for the presence of the `required` field before iterating.

Reply:
Using `value.required?.forEach` now